esp/src/src-react/components/Metrics.tsx

```diff
@@ -197,7 +197,7 @@ export const Metrics: React.FunctionComponent<MetricsProps> = ({
         .baseUrl("")
         .request({
             ScopeFilter: {
-                MaxDepth: 3,
+                MaxDepth: 4,
```

Comment:
@GordonSmith Have you tested the effect of this change on the performance?  Does this submit a wudetails query each time it is called, or is it cached?
I wonder if this is already relatively expensive.

Reply:
I have only tested on local bare metal env...

Reply:
My concern is that this will now cause all the stats from subgraphs to be traversed.  That is going to add a signifcant impact on esp/dali.

I don't think you currently have a way of specifying "fast only" which is what you really need.